src/libopensc/asn1.c

```diff
@@ -1506,7 +1509,7 @@ static int asn1_decode_entry(sc_context_t *ctx,struct sc_asn1_entry *entry,
 	case SC_ASN1_INTEGER:
 	case SC_ASN1_ENUMERATED:
 		if (parm != NULL) {
-			r = sc_asn1_decode_integer(obj, objlen, (int *) entry->parm, 1);
+			r = sc_asn1_decode_integer(obj, objlen, (int *) entry->parm, 0);
```

Comment:
clang-format suggested the following change:


```
-			r = sc_asn1_decode_integer(obj, objlen, (int *) entry->parm, 0);
+			r = sc_asn1_decode_integer(obj, objlen, (int *)entry->parm, 0);
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/asn1.c -->